gc.c

```diff
@@ -3680,9 +3680,15 @@ obj_free(rb_objspace_t *objspace, VALUE obj)
           case imemo_parser_strterm:
             RB_DEBUG_COUNTER_INC(obj_imemo_parser_strterm);
             break;
-          case imemo_callinfo:
+          case imemo_callinfo: {
```

Comment:
Nit: Ruby places the curly bracket on the new line.

```suggestion
          case imemo_callinfo:
          {
```

Reply:
Do we? It's true for function definitions, but `git grep ": {$" | grep case` returns quite a few results (lots of them are Prism today, but there are 58 excluding Prism).

Reply:
In `gc.c` at least, it places the curly brackets on a newline:

https://github.com/ruby/ruby/blob/97564ddf2baa518e4bda5b7a59111943dc13f210/gc.c#L3553-L3554
https://github.com/ruby/ruby/blob/97564ddf2baa518e4bda5b7a59111943dc13f210/gc.c#L3635-L3636
https://github.com/ruby/ruby/blob/97564ddf2baa518e4bda5b7a59111943dc13f210/gc.c#L7007-L7008
https://github.com/ruby/ruby/blob/97564ddf2baa518e4bda5b7a59111943dc13f210/gc.c#L7050-L7051
https://github.com/ruby/ruby/blob/97564ddf2baa518e4bda5b7a59111943dc13f210/gc.c#L7317-L7318

Reply:
It doesn't seem like a convention established in `gc.c` either:

```
$ git grep -n ": {$" gc.c
gc.c:13591:          case T_STRING: {
gc.c:13603:          case T_SYMBOL: {
gc.c:13614:          case T_MOVED: {
gc.c:13618:          case T_HASH: {
gc.c:13657:          case T_DATA: {
gc.c:13680:          case T_IMEMO: {
gc.c:13714:              case imemo_iseq: {
```

Reply:
Good point. I guess there really isn't a convention eh?